backend/article/models.py

```diff
@@ -0,0 +1,30 @@
+"""
+Models of accounts
+"""
+from django.db import models
+from wordlist.models import Phrase
+
+class Article(models.Model):
+  """
+  A class to represent article.
+  :param title: Title of the article
+  :type title: `django.db.models.CharField`
+  :param author: Author of the article
+  :type author: `django.db.models.CharField`
+  :param content: Content of the article
+  :type content: `django.db.models.TextField`
+  :param difficulty: Difficulty of the article
+  :type difficulty: `django.db.models.IntegerField`
+  :param phrases: Phrases of the article
+  :type phrases: `django.db.models.ManyToManyField`
+  :param registered_date: Registered date of the article
+  :type registered_date: `django.db.models.DateTimeField`
+  """
+  title = models.CharField(max_length=200)
+  author = models.CharField(max_length=200)
+  content = models.TextField()
+  difficulty = models.IntegerField(default=5)
+  phrases = models.ManyToManyField(Phrase)
```

Comment:
Instead of m:n relation, maybe 1:n would be more adequate? Since, even if there is a phrase in another article by little chance, it can be understood as a different phrase because context is usually different.